src/Model/Operation/ProductSyncHandler.php

```diff
@@ -462,6 +464,8 @@ private function getDomainUrl(
     protected function getShopwareProducts(array $productIds, SalesChannelContext $context): SalesChannelProductCollection
     {
         $criteria = new Criteria($productIds);
+        $criteria->addAssociation('media');
```

Comment:
Can we use the common criteria within the product helper?

Reply:
Of course. I will do it that way.